projects/packages/search/src/instant-search/class-instant-search.php

```diff
@@ -400,6 +400,63 @@ public function auto_config_overlay_sidebar_widgets() {
 
 		update_option( $widget_opt_name, $widget_options );
 		update_option( 'sidebars_widgets', $sidebars );
+
+		if ( \current_theme_supports( 'block-templates' ) ) {
+			$this->add_search_block_below_header();
+		}
```

Comment:
Could we move this into `auto_config_search`?

Reply:
I put the logic here because search widget is added to theme sidebar in the same function. I refactored the way it works in https://github.com/Automattic/jetpack/pull/22632, where there would be a function for it. And it'll be placed [there](https://github.com/Automattic/jetpack/pull/22632/files#diff-eb54b635de49fca2ee1cdee9bf5ef2015d5eb056ce51338a313f7512a629c01dR364).